pkl-core/src/main/java/org/pkl/core/repl/ReplServer.java

```diff
@@ -83,7 +83,9 @@ public ReplServer(
     this.workingDir = workingDir;
     this.securityManager = securityManager;
     this.moduleResolver = new ModuleResolver(moduleKeyFactories);
-    this.errorRenderer = new VmExceptionRenderer(new StackTraceRenderer(frameTransformer));
+    this.errorRenderer =
+        new VmExceptionRenderer(
+            new StackTraceRenderer(frameTransformer), TextFormatter.create(false));
```

Comment:
`--color` is an arg that's also available to `pkl repl`. So, let's add `color` as a field and constructor arg to `ReplServer`, and the `color` option through in `CliRepl`.

Also (per my other comment): change this to just accept a boolean.

```suggestion
    this.errorRenderer =
        new VmExceptionRenderer(
            new StackTraceRenderer(frameTransformer), color);
```